### DISCUSSION TITLE
Final Milestone Preparation Deadline: 25/01/2021 23.59

### DISCUSSION THREAD

**Discussion by cihatkapusuz** [bounswe/bounswe2020group6#350] [Ideas]
We will carry out our planned tasks on the [Project Plan](https://github.com/bounswe/bounswe2020group6/blob/master/Files/Templates/Project%20Plan1.pdf). The dates given may vary. 

For now, it is decided that the team carry out all remaining tasks, 

### 🔮 The back-end team should implement:
- [x]  endpoints for events
- [x]  update endpoints for notifications
- [x]  semantic search
- [x]  delete user
- [ ] test for endpoints

### 🌺 The front-end  team should implement: 
- [x]  events page
- [x]  notifications improvement
- [x]  password validation 
- [x]  KVKK for signup
- [x]  about us page

### 🤖 The android  team should implement: 
- [x]  events page
- [x]  notification system
- [x]  edit and delete project
- [ ]  delete user
- [x]  home page improvement
- [ ]  password validation
- [ ]  KVKK for signup
- [x]  invitation to a project will improved
- [ ]  downloading files of a project

**Comment by Tubagc** [comment]
Requirements Review

- 1.1.11
- reset password
- 1.1.16
- 1.1.23.3
- 1.1.26.3
- notification to all collaborators -> (x joined to the project)

**Comment by isiktashamza** [comment]
implement change password functionality. 
delete invalid users after a specified time.

**Comment by Tubagc** [comment]
Home page functionality is improved on the mobile. It also recommends users according to their interest areas, universities, and departments.

**Reply by Tubagc** [reply]
The notification system is also added to the mobile. It notifies a user if there is an invitation to the user. Or it notifies a user (the project owner) if there is a request. If there is a new collaborator, all collaborators will be notified. This part is not implemented yet.
